website/docs/policies/index.md

```diff
@@ -46,6 +46,21 @@ Starting with authentik 2022.11.0, the following checks can also be done with th
 -   Check the password hash against the database of [Have I Been Pwned](https://haveibeenpwned.com/). Only the first 5 characters of the hashed password are transmitted, the rest is compared in authentik
 -   Check the password against the password complexity checker [zxcvbn](https://github.com/dropbox/zxcvbn), which detects weak password on various metrics.
 
+### Password Uniqueness Policy
+
+Prevents users from reusing old passwords when changing their own password. The policy offers an option to limit the number many previous passwords to consider during evaluation.
+
+When this policy is bound and enabled to at least one [User write stage](../flow/stages/user_write.md):
+
+1.  authentik compares the hashes of the new password and the old password for a match. The policy check fails and exits if the hashes match.
+2.  authentik copies the hashed form of the user's old password for future comparison
```

Comment:
Suggestion: "When the policy succeeds, the user's current password hash is copied into the password history and the oldest password(s) are removed if the user's password history has more entries than the current depth setting."

Reply:
👍 Much cleaner

We could note if more than one Password Uniqueness Policy is bound & active anywhere in authentik, then the system maintains a history depth equal to the greatest configured depth of all Password Uniqueness Policies. 

e.g. I create 2 copies of a Password Uniqueness Policy. I configure one policy with a depth of `1`, and another with a depth of `10`. authentik will maintain `10` old password hashes.

It's a minor detail and probably only important for someone wanting to fully understand how many old hashes are stored. I'll leave it up to you & @tanberry whether we include that.

Reply:
Ah, I remember you telling me about this. Yes I think we should indeed include this info in the docs. I predict it will avoid a few GitHub Issues in the future, where people ask what-the-heck.

I think that will also help people understand that there can be multiple instances on a policy.